environments/development/calc-corp-ds/process-gpc-data/workflow.yml

```diff
@@ -0,0 +1,33 @@
+dag:
+  repository: moj-analytical-services/airflow-gpc-anomalies-preprocess
+  tag: v0.2.0-rc1
+  retries: 3
+  retry_delay: 150
+  schedule: "0 2 * * *"
+  tasks:
+    unlock_dev:
+      env_vars:
+        R_CONFIG_ACTIVE: "dev"
+      bash_command: "Rscript scripts/unlock_all_blocks.R"
```

Comment:
`bash_command` isn't exposed as an option in our operator. My suggestion would be to make your entrypoint dynamic so it can run a script based on an environment variable

We have an example [here](https://github.com/ministryofjustice/analytical-platform-airflow/blob/main/environments/production/analytical-platform/example/workflow.yml#L7), which is used in [here](https://github.com/moj-analytical-services/analytical-platform-airflow-python-example/blob/main/Dockerfile#L12) and [here](https://github.com/moj-analytical-services/analytical-platform-airflow-python-example/blob/main/src/entrypoint.sh)